guardian/source-android
# Setup publishing

## What does this change?

Prepares the repo for publishing, including defining group and artifact ids. 

Tested this using these steps:
1. Generate a local repo using `gradle publishReleasePublicationToGusourceRepository`
2. Copy the repo to `~/.m2/repository.`
3. Set up `android-news-app` to consume the repository from `mavenLocal`
   * [Branch](https://github.com/guardian/android-news-app/compare/ab/health/fix-sp-dimen-resource...ab/10pc/source-android-integration?w=1)
4. Use the defined typography in a sample screen


Pending for future PRs:
1. Integrate with Sonatype to publish on MavenCentral
2. Work out providing documentation and source in IDE (expect AS to automatically download them from maven central but needs to be checked)
3. Add screenshot tests to the library
4. Update sample app with typography screens
